Doc/library/io.rst

```diff
@@ -769,9 +770,12 @@ than raw I/O does.
 
    .. method:: peek(size=0, /)
 
-      Return bytes from the stream without advancing the position.  At most one
-      single read on the raw stream is done to satisfy the call. The number of
-      bytes returned may be less or more than requested.
+      Return bytes from the current position onwards without advancing the position.
+      At least one byte of data is returned if not at EOF.
+      Return an empty :class:`bytes` object at EOF.
+      At most one single read on the underlying raw stream is done to satisfy the call.
+      The exact number of bytes returned is unspecified
+      (*size* is ignored).
```

Comment:
I would suggest to make it more explicit that the *size* argument is ignored:

The *size* argument is ignored. The number of read bytes depend on the buffer size and the current position in the internal buffer.

Reply:
Updated now